src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1055,8 +1039,8 @@
     <comment>Header for a control to toggle changes in the app title.</comment>
   </data>
   <data name="Profile_SuppressApplicationTitle.HelpText" xml:space="preserve">
-    <value>Use the tab title to override the default title of the tab and suppress any title change messages from the application.</value>
-    <comment>A description for what the "suppress application title" setting does. Presented near "Profile_SuppressApplicationTitle".</comment>
+    <value>Ignore application requests to change the title using OSC 2.</value>

```

Comment:
that may be harder to translate, and there is not a hugely accepted vocabulary about what these are actually called. Technically, "in-band control sequences"... but also OSC is "operating system control", so the "operating system control control sequence". But also, it doesn't use CSI - which is the "Control Sequence Introducer"? It's complicated!

Reply:
idk. i'm open to it!

Reply:
I'd personally write
> Ignore application requests to change the title (OSC 2).

I feel like saying that it's a VT sequence or only OSC 2 doesn't really add anything to the idea of "prevent title changes". Especially since `SetWindowText` is not OSC 2 - it just so happens that we translate it to OSC 2 in ConPTY which is sort of an implementation detail of WT. 🤔 I would personally only add the "OSC 2" in parenthesis at the end, because it might help with google-ing it.

Reply:
Not saying anything related to VT sequences doesn't help with looking it up online - try searching for "OSC 2" and see the results haha. That's why I'd like some sort of extra explanation/context so users can try to search for what it means

Reply:
To be fair, on its own it doesn't find much... but `terminal osc 2` sure does.